problems/euler/multiplesOf3And5/java/jailarie/eulerProblemOne.java

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Maybe add a comment up top indicating what this program is. The class name does indicate that but I still think it doesn't hurt for readability.

Reply:
Good call. I added a short comment. Thanks!